arch/x86/core/prep_c.c

```diff
@@ -17,6 +17,10 @@ extern void x86_64_irq_init(void);
 __pinned_data x86_boot_arg_t x86_cpu_boot_arg;
 #endif
 
+
+
```

Comment:
Nit: lots of new lines here. But only care about those if you need to rebase this PR anyway.